splunk_connector.py

```diff
@@ -178,6 +183,17 @@ def initialize(self):
         if phantom.is_fail(ret_val):
             return self.get_status()
 
+        # Validate sleeptime_in_requests
+        ret_val, self.sleeptime_in_requests = self._validate_integer(self, config.get('sleeptime_in_requests', 1),
+                                                                    consts.SPLUNK_SLEEPTIME_IN_REQUESTS_KEY)
+        if phantom.is_fail(ret_val):
+            return self.get_status()
+
+        # Validte if user has entered more than 120 seconds
+        if self.sleeptime_in_requests > 120:
```

Comment:
do we also need to validate not `< 0`? sleep will raise an exception for negative values